components/BirthdayList.js

```diff
@@ -1,24 +1,38 @@
-import { listOfBirthdays } from "../public/lib/birthdays";
 import styled from "styled-components";
 
-export default function BirthdayList() {
-  const birthdays = listOfBirthdays.map((birthday) => {
-    return (
-      <tbody key={birthday.id}>
-        <tr>
-          <td>{birthday.name}</td>
-          <td>{birthday.ideas.join(", ")}</td>
-          <td>{birthday.birthday}</td>
-        </tr>
-      </tbody>
-    );
-  });
-  return <StyledTable>{birthdays}</StyledTable>;
+export default function BirthdayList({ id, name, birthday, ideas }) {
+  return (
+    <StyledEntry>
+      <StyledName>{name}</StyledName>
+      <StyledIdeas>{ideas.join(", ")}</StyledIdeas>
+      <StyledBirthday>{birthday}</StyledBirthday>
+    </StyledEntry>
+  );
 }
 
-const StyledTable = styled.table`
+const StyledEntry = styled.section`
+  display: grid;
+  grid-template-columns: 20% 60% 20%;
+  grid-template-areas: "a b c";
```

Comment:
I don't think you need this prop. Could be wrong though.

Reply:
Thanks, you're right!